<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addentry</class>
 <widget class="QDialog" name="addentry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>520</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="entrylabel">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>New Entry</string>
   </property>
  </widget>
  <widget class="QLabel" name="vehicleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Vehicle:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="VehiclecomboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="OpNeccesary">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Neccesary for Operation</string>
   </property>
  </widget>
  <widget class="QLabel" name="vehicleLabel_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Maintenance Type</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cosmeticRadioButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cosmetic</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="upkeepRadioButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Upkeep</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="fixingRadioButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>140</y>
     <width>89</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Fixing</string>
   </property>
  </widget>
  <widget class="QLabel" name="notesLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Notes:</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="notesTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>381</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>addentry</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>534</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>addentry</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>540</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>upkeepRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>addentry</receiver>
   <slot>upkeepChecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>164</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fixingRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>addentry</receiver>
   <slot>fixingChecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>326</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cosmeticRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>addentry</receiver>
   <slot>cosmeticChecked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>46</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>36</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>isNeccesary()</slot>
  <slot>upkeepChecked()</slot>
  <slot>fixingChecked()</slot>
  <slot>cosmeticChecked()</slot>
  <slot>notesChanged()</slot>
  <slot>add()</slot>
 </slots>
</ui>
